embedded/remotestorage/s3/s3.go

```diff
@@ -481,39 +512,42 @@ func (s *Storage) Put(ctx context.Context, name string, fileName string) error {
 // Note that due to an asynchronous nature of cloud storage,
 // a resource stored with the Put method may not be immediately accessible.
 func (s *Storage) Exists(ctx context.Context, name string) (bool, error) {
-	if strings.HasPrefix(name, "/") || strings.HasSuffix(name, "/") {
-		return false, ErrInvalidArguments
-	}
-
-	url, err := s.originalRequestURL(name)
+	entries, _, err := s.scanObjectNames(ctx, name, 1)
```

Comment:
wondering why `Exists` doesn't accept names or prefixes with slashes but `Get` method accept it